.gitignore

```diff
@@ -1,4 +1,4 @@
 # Created by .ignore support plugin (hsz.mobi)
 .idea/
-bower_components
-.DS_Store
+bower_components/
+*.DS_Store
```

Comment:
@cxoltero 
With this update to `.gitignore`, you'll now have to manually remove the `js/.DS_Store` and `css/.DS_Store` files and commit their deletion.